app/stylesheet/legacy/alerts.scss

```diff
@@ -68,11 +68,11 @@
         .fixed-col {
           height: 26px;
 
-          @media (min-width: $screen-sm) {
+          @media (min-width: 728px) {
```

Comment:
why hardcode the value instead of using a variable?  if we ever need to change it, we will have to update it in many places.  Maybe I just do not understand the rationale for this type of change

Reply:
Yes i was trying out few things . It is still WIP. i will cleanup code once it is done.